CONTRIBUTING.md

```diff
@@ -15,6 +15,15 @@ Execute the following shell command:
 $ dotnet build
 ```
 
+Code formatter
+--------------
+
+We use [Fantomas][fantomas] to format F# code automatically. To install tool locally use command:
+
+```console
+$ dotnet tools restore
```

Comment:
It would also be useful to leave a note how to reformat all the sources in the solution. If this is possible, of course.